shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/jaxb/sql/loader/SQLCasesLoader.java

```diff
@@ -117,7 +117,7 @@ private String getLiteralSQL(final String sql, final List<?> parameters) {
         if (null == parameters || parameters.isEmpty()) {
             return sql;
         }
-        return String.format(sql.replace("%", "$").replace("?", "%s"), parameters.toArray()).replace("$", "%").replace("%%", "%").replace("'%'", "'%%'");
+        return String.format(sql.replace("%", "$$").replace("?", "%s"), parameters.toArray()).replace("$$", "%").replace("%%", "%").replace("'%'", "'%%'");
```

Comment:
@lwtdev 

What is the purpose of such a change?

Reply:
@jingshanglu `$` is used to temporarily replace `%`, but in [MySQL](https://dev.mysql.com/doc/refman/5.7/en/json.html#json-path-syntax), it used to be the Path syntax of JSON, representing a JSON document.
The test case is not working properly.

Maybe using a non-printable character here is better than "$$ "?

Reply:
Add a replace method to replace sql parameters.